src/pages/components/RevokeActionsButton.js

```diff
@@ -4,31 +4,77 @@ import { Button, Dialog } from '@digicatapult/ui-component-library'
 
 import ReasonsPopup from './ReasonsPopup'
 
+// ReasonsViewPopup(imports)
+import { Section } from '@digicatapult/ui-component-library'
+// import WarningSignSvg from '../../assets/images/warning-sign.svg'
+import WarningSignSvg from '../../assets/images/warning-sign-icon.svg'
+
+// ReasonsViewPopup
+const ReasonsViewPopup = function ({ handleCancel }) {
+  return (
+    <>
+      <Section
+        headingLevel={1}
+        title=""
+        background="#27847a"
+        headingSize="0em"
+        padding="10px 141px"
+      >
+        <DivWarning>
+          <TitleWarning>
+            This hydrogen certificate has been revoked. <br />
+            <SubtitleWarning>
+              This certificate has been revoked by the regulator and its status
+              will be reflected on the blockchain.
```

Comment:
Don't mention blockchain.

Just 'this certificate has been revoked by the regulator'

Reply:
Done, fixed.